addons/mass_mailing/static/src/scss/mass_mailing.wysiwyg.scss

```diff
@@ -23,6 +23,22 @@
     top: 0;
     z-index: $zindex-dropdown - 1;
 
+    .o_we_external_history_buttons {
+        display: flex;
+        margin: auto 160px auto auto;
+
+        .btn {
+            align-items: center;
+            width: 24px;
+            height: 24px;
+            background-color: #337ab7;
+            border: 1px solid #2e6da4;
+            border-radius: 4px;
+            padding: 0;
+            margin-left: 5px;
+        }
```

Comment:
could we make something like 
```scss
@mixin email_designer_btn {
    align-items: center;
    width: 24px;
    height: 24px;
    background-color: #337ab7;
    border: 1px solid #2e6da4;
    border-radius: 4px;
    padding: 0;
    margin-left: 5px;
}
``` 
and `@include` it in the two `.btn` spots?

Or simply make it a `btn-email-designer` class and use it directly it both places?